.github/workflows/integration.yml

```diff
@@ -99,12 +94,12 @@ jobs:
         options: --health-cmd="curl localhost:9200/_cluster/health?wait_for_status=yellow&timeout=60s" --health-interval=10s --health-timeout=5s --health-retries=3
     steps:
       - uses: actions/checkout@v4
-      - name: M2 Integration Tests with Magento 2 Version 2.4.5-p6 (PHP 8.1)
-        uses: extdn/github-actions-m2/magento-integration-tests/8.1@master
+      - name: M2 Integration Tests with Magento 2 Version 2.4.4-p9 (PHP 8.1)
+        uses: extdn/github-actions-m2/magento-integration-tests/7.4@master
```

Comment:
This is the wrong PHP version. However, according to https://github.com/customgento/m2-github-actions/pull/6/files#diff-82454b2fc887e9985b9348b8f3bca03d2f839645a4c3ba1f3f850014d7da5c8bL75, this Magento version most probably won't work with PHP 8.1.

Reply:
I checked for like "module-mass-update-tier-prices-m2". It worked without any issue.

Reply:
Where did you test this? Could you please add a link here?

Reply:
here is the link please: https://github.com/customgento/module-mass-update-tier-prices-m2/pull/44

Reply:
here you see https://github.com/customgento/module-mass-update-tier-prices-m2/pull/44 that 2.4.4-p8 PHP 8.1 is failing as with same error previously reported https://github.com/magento/magento2/issues/35292.